eox_tagging/edxapp_accessors.py

```diff
@@ -55,8 +57,7 @@ def get_course_enrollment(**kwargs):
 
     enrollment, _ = get_enrollment(username=username, course_id=course_id)
     try:
-        # Temporary method to get CourseEnrollment objects. This should be a backend.
-        from student.models import CourseEnrollment
+        # Temporary way of getting CourseEnrollment objects.
```

Comment:
why temporary ?

Reply:
Using the model from the platform shouldn't be the best way right?

Reply:
right, you remove that behavior, so my question is why we need this comment, what is the different between the enrollment from `get_enrollment(username=username, course_id=course_id)` and `CourseEnrollment.objects.get(user_id=user.id, course_id=course_id)`?  why don't just use  `get_enrollment(username=username, course_id=course_id)`  ?

Reply:
We need to have the object that we want to tag to be able to use generic foreign key [see the example here](https://docs.djangoproject.com/en/1.11/ref/contrib/contenttypes/#django.contrib.contenttypes.fields.GenericForeignKey), and get_enrollments returns a dictionary with the enrollment information. Because we need the enrollment object, we have to use CourseEnrollment.objects.get

Reply:
hence you can remove the line 54 ? and change the line 60 since that exception will catch in https://github.com/eduNEXT/eox-tagging/blob/62371e9853497318b5d3fae63a70a426ef62bf5c/eox_tagging/edxapp_wrappers/backends/enrollment_i_v1.py#L11

Reply:
Right!